todo-vue/src/components/Card.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <li id="wrapper">
+    <div id="title-container">
+      <form v-if="isEditing" @submit="handleSubmit">
+        <input type="text" id="edit-todo-input" />
+      </form>
+      <h3 id="title" :class="{ done: isDone }" v-else>
+        {{ title }}
+      </h3>
+      <div id="created-at">{{ createdAt.toLocaleDateString() }}</div>
+    </div>
+    <div id="control-container">
+      <v-icon class="icon" @click="toggleEdit">mdi-pencil</v-icon>
+      <v-icon class="icon" @click="toggleDone">mdi-check</v-icon>
+      <v-icon class="icon" @click="handleDelete">mdi-close</v-icon>
+    </div>
+  </li>
+</template>
+
+<script>
+export default {
+  name: "CardComponent",
+  data() {
+    return { createdAt: new Date(), isEditing: false, isDone: false };
+  },
+  props: {
+    title: String,
+    id: Number,
+  },
```

Comment:
```suggestion
  props: {
    title: { type: String, required: true },
    id: { type: Number, required: true },
  },
```
이렇게 필수 여부를 결정할 수도 있습니다.

그리고 type에 `String`과 `Number`를 쓰고 있는데,
어떤 방식으로 타입을 검증하는지 혹시 유추해볼 수 있을까요?